src/rebar_prv_lock.erl

```diff
@@ -67,6 +66,7 @@ build_locks(State) ->
          rebar_app_info:dep_level(Dep)}
      end || Dep <- AllDeps, not(rebar_app_info:is_checkout(Dep))].
 
-info_checkout_deps(Checkouts) ->
+info_checkout_deps(Checkouts0) ->
+    Checkouts = lists:usort(Checkouts0),
```

Comment:
Sometimes the checkouts list will have duplicated dep names, this should also help remove duplicated messages.